src/build-time-render/Renderer.ts

```diff
@@ -3,6 +3,31 @@ const { JSDOM, ResourceLoader } = jsdom;
 
 export type Renderer = 'puppeteer' | 'jsdom';
 
+class CustomResourceLoader extends ResourceLoader {
+	private _requestCount = 0;
+
+	get requestCount() {
+		return this._requestCount;
+	}
+
+	fetch(url: string, options: any) {
+		if (options.element && options.element.localName === 'iframe') {
+			return null;
+		}
+		this._requestCount++;
+		const response = super.fetch(url.replace(/#.*/, ''), options);
+		response.then(
```

Comment:
Would `response.finally` work here?

Reply:
Yup I think it would have, didn’t change that tho.